src/celengine/rendcontext.cpp

```diff
@@ -772,9 +773,8 @@ GLSLUnlit_RenderContext::makeCurrent(const Material& m)
         return;
 
     prog->use();
-    prog->MVPMatrix = (*projectionMatrix) * (*modelViewMatrix);
-    if (usePointSize)
-        prog->ModelViewMatrix = *modelViewMatrix;
```

Comment:
@375gnu  also was not sure about this, where ModelViewMatrix might not be set

Reply:
in the current code we don't need `ModelViewMatrix` every time, but to simplify your changes you can set it always.